jmx/pom.xml

```diff
@@ -95,5 +95,17 @@
             </dependencies>
         </profile>
     </profiles>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <argLine>-Djdk.attach.allowAttachSelf=true</argLine>
```

Comment:
This will overwrite all the other options already specified. Maybe that’s ok here.

Reply:
Right.. Alternatively we can disable `testJava9Agent` on CI.
Other ideas?

Reply:
Maybe we should add this option unconditionally in Airbase? For now, I think overriding here is fine.

Reply:
https://github.com/airlift/airbase/pull/216